### PR TITLE
Fix/some problems with rerender

### PR DESCRIPTION
**Pull request by IvanUglovets**

### CODE REVIEW

File: packages/frontapp/src/app/app.tsx [hunk 1]
```diff
@@ -1,26 +1,13 @@
-import { FC, useCallback, useState } from 'react';
+import React, { FC, useCallback, useState } from 'react';
```
Comment (x8er) [inline packages/frontapp/src/app/app.tsx:1]: no need to import whole React library

File: packages/frontapp/src/app/components/MainComponent/index.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
+import React, { FC, SetStateAction, useCallback } from 'react';
```
Comment (x8er) [inline packages/frontapp/src/app/components/MainComponent/index.tsx:1]: the same as above. You can just import the `memo` method and use it

File: packages/frontapp/src/app/components/NotificationIcon/index.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { FC } from 'react';
+import React, { FC } from 'react';
```
Comment (x8er) [inline packages/frontapp/src/app/components/NotificationIcon/index.tsx:1]: the same as above

File: packages/frontapp/src/app/components/SearchWrapper/index.tsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { Dispatch, FC, SetStateAction } from 'react';
+import React, { FC } from 'react';
```
Comment (x8er) [inline packages/frontapp/src/app/components/SearchWrapper/index.tsx:1]: the same as above

File: packages/frontapp/src/app/components/Sidebar/index.tsx [hunk 5]
```diff
@@ -1,12 +1,12 @@
-import { Dispatch, FC, SetStateAction } from 'react';
+import React, { FC } from 'react';
```
Comment (x8er) [inline packages/frontapp/src/app/components/Sidebar/index.tsx:1]: the same as above

**Review by x8er — CHANGES_REQUESTED** [review]

**Review by x8er — APPROVED** [review]

**Review by lexussuxel — APPROVED** [review]

**Review by x8er — APPROVED** [review]

**Pull request merged**